src/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -379,3 +382,11 @@ def get_random_available_port():
             return port
         finally:
             tcp.close()
+
+    @staticmethod
+    def get_flower_machine(env):
+        return next(
+            celery_host
+            for celery_host, celery_options_by_queue in env.app_processes_config.celery_processes.items()
+            if 'flower' in celery_options_by_queue
+        )
```

Comment:
This dense little nugget searches for the [flower](https://github.com/dimagi/commcare-cloud/blob/e94fb7a46bf0723095a51fbbe30ab7cda86befad/environments/production/app-processes.yml#L69) entry in `app-processes.yml` in order to find which machine it's declared to run on.